galsim/hsm.py

```diff
@@ -127,9 +130,17 @@ def image_bounds(self): return BoundsI(self._data.image_bounds)
     @property
     def moments_status(self): return self._data.moments_status
 
+    @property
+    def observed_e1(self):
+        return self._data.observed_e1
+
+    @property
+    def observed_e2(self):
+        return self._data.observed_e2
+
     @property
     def observed_shape(self):
-        return Shear(e1=self._data.observed_e1, e2=self._data.observed_e2)
+        return Shear(e1=self.observed_e1, e2=self.observed_e2)
```

Comment:
@erykoff wanted this to use `galsim._Shear` here, but that requires conversion from `e` to `g` before we invoke `_Shear` and might not be all the worthwhile.

Reply:
If you want to optimize this too, you could try:
```
e = self._data.observed_e1 + 1j*self._data.observed_e2
g = e / (1 + np.sqrt(1.-np.abs(e)**2))
return _Shear(g)
```
I suspect that's a little faster, but might just be nibbling at the edges.

Reply:
In the LSST DM code base, the reason to get `observed_shape` was because there was no way to get `observed_e1` directly up until now. Now that we can access them directly, I honestly don't think using `_Shear` is necessary.